API/Controllers/ProjectController.cs

```diff
@@ -228,7 +228,7 @@ public async Task<IActionResult> DeleteProject(int projectId)
                     Detail = "The user is not allowed to delete the project.",
                     Instance = "D0363680-5B4F-40A1-B381-0A7544C70164"
                 };
-                return BadRequest(problem);
+                return Unauthorized(problem);
```

Comment:
The 401 Unauthorized status should be used when the user is not authenticated / signed in.
In this case, the user is authenticated but does not have the right permissions. The return status should be 403 Forbidden.

```suggestion
                return Forbidden(problem);
```